### PR TITLE
Fix a compile issue caused by vcpkg providing a symbol at link time

### PR DESCRIPTION
**Pull request by dzenanz**
See https://github.com/InsightSoftwareConsortium/ITK/issues/4552.

### CODE REVIEW

File: CMakeLists.txt
[the remark this answers is not in the document]
Comment (dzenanz) [reply CMakeLists.txt:1]: Then please suggest an alternative.

[the remark this answers is not in the document]
Comment (vfonov) [reply CMakeLists.txt:1]: I don't have a mac to try to run this on.

[the remark this answers is not in the document]
Comment (gdevenyi) [reply CMakeLists.txt:1]: This looks to be the standard fix on the web for this failure and I haven't seen anything about Macs failing from it.

[the remark this answers is not in the document]
Comment (dzenanz) [reply CMakeLists.txt:1]: Please merge, so we can close https://github.com/InsightSoftwareConsortium/ITK/issues/4552.

**Pull request merged**